packages/tools/pluggable-widgets-tools/configs/rollup.config.js

```diff
@@ -132,6 +132,7 @@ export default async args => {
             },
             treeshake: { moduleSideEffects: false },
             plugins: [
+                url(),
```

Comment:
Is this one necessary?

Reply:
Yes, otherwise we get the following error: `Error: Unexpected token (Note that you need plugins to import files that are not JavaScript)`